src/main/java/apoc/temporal/TemporalProcedures.java

```diff
@@ -14,6 +13,67 @@
 
 public class TemporalProcedures
 {
+    private static final Map<String, DateTimeFormatter> ISO_DATE_FORMAT = new HashMap<>();
+
+    static {
```

Comment:
Mettete un commento tipo: "Static default formatters from Java ISO and Elasticsearch patterns"